openedx/features/caliper_tracking/tests/expected/edx.team.searched.json

```diff
@@ -0,0 +1,41 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "Searched",
+  "actor": {
+    "id": "http://localhost:18000/u/edx",
+    "name": "edx",
+    "type": "Person"
+  },
+  "eventTime": "2018-12-26T12:57:21.978Z",
+  "extensions": {
+    "extra_fields": {
+      "accept_language": "en-US,en;q=0.9",
+      "agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.80 Safari/537.36",
+      "event_source": "server",
+      "event_type": "edx.team.searched",
+      "host": "8c3b43aa7b1b",
+      "ip": "172.18.0.1",
+      "page": null,
+      "path": "/api/team/v0/teams/",
+      "session": "8b5aaa43369a0d612918005f1a65f27f",
+      "user_id": 2
+    }
+  },
+  "id": "urn:uuid:d4618c23-d612-4709-8d9a-478d87808067",
+  "object": {
+    "extensions": {
+      "course_id": "course-v1:edx+CS0110+2014_T",
+      "number_of_results": 0,
+      "org_id": "edx",
+      "search_text": "hello",
+      "topic_id": "Topic1ID"
+    },
+    "id": "http://localhost:18000/courses/course-v1:edx+CS0110+2014_T/teams/",
+    "type": "Frame"
```

Comment:
What is the reason behind using `Frame` here?

Reply:
done